database/migrations/2020_11_11_192121_create_companies_table.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->string('name')->unique();
             $table->foreignId('owner_id')->references('id')->on('users');
             $table->string('logo_path')->nullable();
+            $table->unsignedBigInteger('usable_fund')->default(0);
```

Comment:
Er zijn mensen die zeggen dat het beter is om een nieuwe migration aan te maken, met een ALTER TABLE statement erin. Voor nu vind ik het niet zo erg, maar weet dat er een andere manier is om dit te doen die door veel mensen verkozen wordt boven dit

Reply:
Klopt. Dacht dat het nu een beetje onnodig is omdat we toch mock data hebben en het zo seeden. Wanneer je een product op de markt zet is het natuurlijk een heel ander verhaal.